highlight.io/components/QuickstartContent/self-host/self-host.tsx

```diff
@@ -9,21 +9,46 @@ export const SelfHostContent: QuickStartContent = {
 		{
 			title: 'Prerequisites',
 			content:
-				'Before we get started, you should have the latest version of [Docker](https://docs.docker.com/engine/install/) (19.03.0+) ' +
+				'Before we get started, you should install [Go](https://go.dev/) (1.20), [Node.js](https://nodejs.org/en) (18), and [yarn](https://yarnpkg.com/getting-started/install) (v3+).' +
+				'You should have the latest version of [Docker](https://docs.docker.com/engine/install/) (19.03.0+) ' +
 				'and [Git](https://git-scm.com/downloads) (2.13+) installed. ' +
 				'For a local hobby deploy, we suggest [configuring docker](https://docs.docker.com/desktop/settings/mac/#resources) ' +
 				'to use at least 8GB of RAM, 4 CPUs, and 64 GB of disk space.',
 			code: [
 				{
 					language: 'bash',
-					text: `$ docker --version
+					text: `$ go version
```

Comment:
One thing I wanted to raise about this code snippet is I don't think the copy/paste button functionality will work appropriately for the terminal. But looks like this was the existing case, so not blocking

Reply:
good point, makes sense to break this up into separate code blocks and make the copy icon only copy the bash command. making that change